package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "backstage-modal": "^0.3.2",
     "classnames": "^2.2.5",
-    "draft-js": "0.10.5",
+    "draft-js": "0.11.0",
```

Comment:
Why not 0.11.4?

Reply:
Done. Didn't want to jump too many versions but tested with 0.11.4 and seems ok.